src/main/java/net/reini/rabbitmq/cdi/ConnectionListener.java

```diff
@@ -1,34 +1,32 @@
 package net.reini.rabbitmq.cdi;
 
 import com.rabbitmq.client.Connection;
+import java.util.EventListener;
 
 /**
- * A connection listener is used by a connection factory to notify clients about a change in
- * connection state.
- * 
+ * A connection listener is used by a connection factory to notify clients about a change in connection state.
```

Comment:
No real change... (seems to be a formatting difference between your IDE and Eclipse here)

Reply:
Yes i use idea i think there is some info on our internal company wiki how to get it equal to eclipse. I have a look tomorrow

Reply:
could you send me your eclipse formatting preferences? i can use the eclipse formatter from inside idea if i have the correct settings